### ISSUE TITLE
TEAM Engine source code has mixed line endings

### ISSUE THREAD

**Issue by dgtombs** [opengeospatial/teamengine#358]
The TEAM Engine source code has multiple UNIX/Windows line ending issues:

* Some source files have LF endings in the repository (e.g., CtlEarlReporter.java) while most have CRLF.
* Some source files even have mixed line endings themselves (e.g., XMLValidatingParser.java).
* If Git's `core.autocrlf` is set to true (common on Windows), *.sh files check out with CRLF line endings, which breaks them.

### Suggestion

1. Use a `.gitattributes` file like [this one](https://github.com/alexkaratarakis/gitattributes/blob/master/Java.gitattributes).
2. Use `git add --renormalize .` to normalize line endings in the repository. (Fuller instructions in [the Git docs](https://git-scm.com/docs/gitattributes#_end_of_line_conversion).)

Since this procedure will modify every line in quite a few files, it should be done at a good stopping point while people are not working on other commits.

**Comment by LesleyBrownLiberty** [comment]
I was working with dgtombs and have fixes for another issue completed [357](https://github.com/opengeospatial/teamengine/issues/357).  However, those changes were on a branch including these changes with the assumption this pull request would be trivial.  
Would you prefer I proceed with pull request for 357 (that includes this issue) or wait for this issues pull request be approved?

**Comment by dstenger** [comment]
Thank you for your contribution.
However, there is no pull request for this issue. Maybe, you are referencing the pull request for issue https://github.com/opengeospatial/teamengine/issues/359 (please be aware that there is a merge conflict in this PR; so, please do a pull/merge of current master if possible)?

**Comment by LesleyBrownLiberty** [comment]
My mistake.  It appears issue [#359](https://github.com/opengeospatial/teamengine/issues/359) includes changes for this issue as well.  I will take the action to review 359 and update with the current master.

**Comment by dstenger** [comment]
Thank you!

For us it is fine if you already create both pull requests.
Please add a note in pull request including code changes of the other pull request. By that, we know that the other pull request must be merged first.
